docs/PlotCreating.md

```diff
@@ -11,7 +11,7 @@ Special plotting script is responsible for creating benchmark plots. It is a Pyt
 
 These dependencies are fetched automatically during CMake configure.
 
-This script looks through the directory `tmp` and for every subdirectory, that contains results produced by [the global runner](./GlobalRunner.md), it creates graphs and stores them in the folder `tmp/graphs`. Two kinds of graphs are available: static PNG files and dynamic [plotly](https://plot.ly/python/) HTML files.
+By default, this script looks through the directory `tmp` and for every its subdirectory, that contains results produced by [the global runner](./GlobalRunner.md), it creates graphs and stores them into the folder `tmp/graphs`. Two kinds of graphs are available: static PNG files and dynamic [plotly](https://plot.ly/python/) HTML files. Also, it saves two JSON files that are contain all test sizes (`graphs_index.json`) and all collected plotted info, e.g. times, tool names etc. (`plot_data.json`). If 
```

Comment:
Actually absence of "its" and "in" rather than `into` were already correct here.

Reply:
Reverted